actionners/kubernetes/drain/drain.go

```diff
@@ -92,47 +100,53 @@ func Action(action *rules.Action, event *events.Event) (utils.LogLine, *model.Da
 
 	for _, p := range pods.Items {
 		wg.Add(1)
+		p := p
 		go func() {
 			defer wg.Done()
-
-			ownerKind, err := kubernetes.GetOwnerKind(p)
+			var ownerKind string
+			ownerKind, err = kubernetes.GetOwnerKind(p)
 			if err != nil {
 				utils.PrintLog("warning", utils.LogLine{Message: fmt.Sprintf("error getting pod '%v' owner kind: %v", p.Name, err)})
 				otherErrorsCount++
 				return
 			}
 
 			switch ownerKind {
-			case "DaemonSet":
+			case utils.DaemonSetStr:
 				if config.IgnoreDaemonsets {
 					ignoredPodsCount++
 				}
-			case "StatefulSet":
+			case utils.StatefulSetStr:
 				if config.IgnoreStatefulSets {
 					ignoredPodsCount++
 				}
-			case "ReplicaSet":
-				replicaSetName, err := kubernetes.GetOwnerName(p)
+			case utils.ReplicaSetStr:
+				var replicaSetName string
+				replicaSetName, err = kubernetes.GetOwnerName(p)
 				if err != nil {
 					utils.PrintLog("warning", utils.LogLine{Message: fmt.Sprintf("error getting pod owner name: %v", err)})
```

Comment:
```suggestion
					utils.PrintLog("warning", utils.LogLine{Message: "action", Output: fmt.Sprintf("error getting pod owner name: %v", err)})
```

Reply:
The Message field is used to give the context (init, action, notification, ...), use Output for the real "message"